tools/StaticAnalysis/Exceptions/Az.Network/BreakingChangeIssues.csv

```diff
@@ -501,3 +501,6 @@
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.GetInterfaceEndpointCommand","Get-AzInterfaceEndpoint","0","1060","The parameter set 'GetByNameParameterSet' for cmdlet 'Get-AzInterfaceEndpoint' is no longer the default parameter set.","Change the default parameter for cmdlet 'Get-AzInterfaceEndpoint' back to 'GetByNameParameterSet'."
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.GetInterfaceEndpointCommand","Get-AzInterfaceEndpoint","0","2000","The cmdlet 'Get-AzInterfaceEndpoint' no longer supports the parameter 'ResourceId' and no alias was found for the original parameter name.","Add the parameter 'ResourceId' back to the cmdlet 'Get-AzInterfaceEndpoint', or add an alias to the original parameter name."
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.GetInterfaceEndpointCommand","Get-AzInterfaceEndpoint","0","1050","The parameter set 'GetByResourceIdParameterSet' for cmdlet 'Get-AzInterfaceEndpoint' has been removed.","Add parameter set 'GetByResourceIdParameterSet' back to cmdlet 'Get-AzInterfaceEndpoint'."
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzureNetworkProfileContainerNetworkInterfaceConfigCommand","New-AzContainerNicConfig","0","3030","The generic type for 'property ContainerNetworkInterfaces' has been changed from 'System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSContainerNetworkInterface]' to 'System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSResourceId]'. ","Change the generic type for 'property ContainerNetworkInterfaces' back to 'System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSContainerNetworkInterface]'."
```

Comment:
What are these breaking changes?  There should be no breaking change release

Reply:
At some point AutoMapper started to fail (possible there was recent version update) mapping this model. I suspect the problem there was circular reference, so placing just Id instead of a whole resource (which in fact maps to API specs well) resolved the issue.

Last paragraph [in this comment](https://github.com/Azure/azure-powershell/pull/9501#issuecomment-505834228) contains related links.

Reply:
This is a breaking change - you need to figure out how to make the old type work, or create a new type that has the same properties - replacing with a type with fewer properties is not an acceptable option